lib/opensusebasetest.pm

```diff
@@ -166,7 +166,7 @@ sub investigate_yast2_failure {
         "Could not load icon|Couldn't load pixmap"         => undef             # Detecting missing icons
     );
     for my $y2log_error (keys %y2log_errors) {
-        if (my $y2log_error_result = script_output 'grep -B 3 -E "' . $y2log_error . '" /var/log/YaST2/y2log || true') {
+        if (my $y2log_error_result = script_output 'zgrep -B 3 -E "' . $y2log_error . '" /var/log/YaST2/y2log* || true') {
```

Comment:
is it present `zgrep` in all system we want to apply this? I always found some issue in the past with that compression tools.

Reply:
I was not sure if that is the case too, but it's available in the SLE 15 medium. I guess I will introduce fallback mechanism here.

Reply:
I checked within a SLE15SP1 container and I could find "grep" being preinstalled but not "zgrep" so I would also suggest the fallback. However, to make this PR easier: Could we just leave out that change for now and discuss in a dedicated PR?

Reply:
Makes sense, dropped that commit from this PR